charts/security-apps/values.yaml

```diff
@@ -81,7 +81,7 @@ vault:
   # -- Chart
   chart: "vault"
   # -- [vault Helm chart](https://github.com/hashicorp/vault-helm) version
-  targetRevision: "0.19.*"
+  targetRevision: "0.20.1"
```

Comment:
i've always held back on not using `x.y.*` before we don't have chart version bumping automated ala  #218 

but i'm all for changing that asap :+1: